dom0-updates/qubes-dom0-update

```diff
@@ -267,11 +266,18 @@ elif [ -f /var/lib/qubes/updates/repodata/repomd.xml ]; then
             dnf "${YUM_OPTS[@]}" $YUM_ACTION
         fi
     fi
-    dnf -q check-update && qvm-features dom0 updates-available ''
+    if dnf -q check-update; then
+        if ! qvm-features dom0 updates-available ''; then
+            echo "*** WARNING: cannot set feature 'updates-available'" >&2
+        fi
+    fi
 else
-    qvm-features dom0 updates-available ''
-    echo "No updates available" >&2
-    if [ "$GUI" == "1" ]; then
-        zenity --info --title='Dom0 updates' --text='No updates available'
+    if ! qvm-features dom0 updates-available ''; then
+        echo "*** WARNING: cannot set feature 'updates-available'" >&2
+    else
+        echo "No updates available" >&2
```

Comment:
This part should happen even if qvm-features fails.

Reply:
I adjusted the commit.